apps/teams-test-app/src/App.tsx

```diff
@@ -17,7 +17,12 @@ const getOriginsParam = urlParams.has('origins') && urlParams.get('origins') ? u
 const validMessageOrigins: string[] | undefined = getOriginsParam ? getOriginsParam.split(',') : undefined;
 
 // This is added for custom initialization when app can be initialized based upon a trigger/click.
-if (!urlParams.has('customInit') || !urlParams.get('customInit')) {
+if (
+  !urlParams.has('customInit') ||
```

Comment:
nit: It would be better to extract this into a separate function, perhaps named checkUrlParam(urlParam: string) for all the url param checks . This function could also be reused on Line 36, 37 and 38 of this file. The benefit is that it would make the code cleaner and more maintainable.

Reply:
It is unnecessary because most of them would be removed and after remove, the function you suggested will only be written once here for `skipInit`.

Reply:
Also suppose you have a function checking as `!A || !B`, `A && B` is not equivalent to say `not !A || !B` because `A && !B` is a sufficient scenario for `not !A || !B`